docs/data/flink_ml.yml

```diff
@@ -15,11 +15,24 @@
 # specific language governing permissions and limitations
 # under the License
 
+2.3:
+  name: "Apache Flink ML 2.3.0"
+  source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-ml-2.3.0/flink-ml-2.3.0-src.tgz"
+  source_release_asc_url: "https://downloads.apache.org/flink/flink-ml-2.3.0/flink-ml-2.3.0-src.tgz.asc"
+  source_release_sha512_url: "https://downloads.apache.org/flink/flink-ml-2.3.0/flink-ml-2.3.0-src.tgz.sha512"
+  python_source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-ml-2.3.0/apache-flink-ml-2.3.0.tar.gz"
+  python_source_release_asc_url: "https://downloads.apache.org/flink/flink-ml-2.3.0/apache-flink-ml-2.3.0.tar.gz.asc"
+  python_source_release_sha512_url: "https://downloads.apache.org/flink/flink-ml-2.3.0/apache-flink-ml-2.3.0.tar.gz.sha512"
+  compatibility: ["1.17.*"]
+
 2.2:
   name: "Apache Flink ML 2.2.0"
   source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-ml-2.2.0/flink-ml-2.2.0-src.tgz"
   source_release_asc_url: "https://downloads.apache.org/flink/flink-ml-2.2.0/flink-ml-2.2.0-src.tgz.asc"
   source_release_sha512_url: "https://downloads.apache.org/flink/flink-ml-2.2.0/flink-ml-2.2.0-src.tgz.sha512"
+  python_source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-ml-2.2.0/apache-flink-ml-2.2.0.tar.gz"
+  python_source_release_asc_url: "https://downloads.apache.org/flink/flink-ml-2.2.0/apache-flink-ml-2.2.0.tar.gz.asc"
+  python_source_release_sha512_url: "https://downloads.apache.org/flink/flink-ml-2.2.0/apache-flink-ml-2.2.0.tar.gz.sha512"
   compatibility: ["1.15.*"]
 
 2.1:
```

Comment:
Should we delete the information of ml-2.0 and ml-2.1 as other flink projects did?

Reply:
Good point. I have updated the PR as suggested.